neurodsp/sim/periodic.py

```diff
@@ -303,3 +310,15 @@ def get_burst_samples(is_oscillating, fs, freq):
     bursts = np.repeat(is_oscillating, n_samples_cycle)
 
     return bursts
+
+
+def _check_tiling(fs, freq):
+    """Check if tiling can produce full oscillations."""
```

Comment:
I'm not sure 'full oscillations' is totally obvious, so perhaps:
"Check if tiling will produce an integer number of cycles for the simulated oscillation."